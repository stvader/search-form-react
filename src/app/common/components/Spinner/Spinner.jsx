import React from 'react';
import { CircularProgress } from '@material-ui/core';

import s from './spinner.module.scss';

const Spinner = () => (
  <div className={s.wrapper}>
    <CircularProgress />
  </div>
);

export default Spinner;
