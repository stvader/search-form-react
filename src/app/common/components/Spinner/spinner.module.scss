.wrapper {
  margin: 10px;
}

@media screen and (min-width: 750px) {
  .wrapper {
    margin: 30px;
  }
}