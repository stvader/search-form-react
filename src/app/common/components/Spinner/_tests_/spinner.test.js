import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { CircularProgress } from '@material-ui/core';

import Spinner from '../Spinner';

configure({ adapter: new Adapter() });

describe('Spinner', () => {
  it('Spinner render', () => {
    const component = shallow(<Spinner />);

    const spinner = component.find(CircularProgress);

    expect(component).toMatchSnapshot();
    expect(spinner).toBeDefined();
  });
});
