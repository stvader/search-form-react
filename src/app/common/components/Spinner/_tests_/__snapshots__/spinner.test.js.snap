// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Spinner render 1`] = `ShallowWrapper {}`;
